CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+
+- Keep local imports together with no newline between them.
+- Add `_+` as an ignore pattern for `no-unused-vars`.
```

Comment:
Why?

Reply:
- We already use `_` and `__` as a convention for unused variables.
- There's no clear gain by separating with newlines local imports from the current dir and the parent dir. This PR makes our linter separate imports by their origin type: native, external, internal and local.

Reply:
(and keeping local imports together makes it easier sell this kind of linting config for the majority of devs that dislike extra new-lines 🤷‍♂)